bower.json

```diff
@@ -35,7 +35,7 @@
     "jquery-pjax": "1.9.6",
     "jquery-validation": "1.15.1",
     "magnific-popup": "https://github.com/wet-boew/Magnific-Popup.git#1.0.0+keyboard_trap_fix",
-    "openlayers": "https://github.com/wet-boew/openlayers-dist.git#2.13.1",
+    "openlayers": "https://github.com/wet-boew/ol3.git#3.17.2",
```

Comment:
@nschonni I can reference https://github.com/openlayers/openlayers.git#3.17.1 but how do I npm build it after npm install? I need the distribution lib.